packages/explorer/src/common.ts

```diff
@@ -1,5 +1,7 @@
 import { anvil, garnet, redstone } from "viem/chains";
 
+export const internalMUDNamespaces = ["world", "store", "metadata"];
```

Comment:
```suggestion
export const internalNamespaces = ["world", "store", "metadata"];
```
(we're in MUD and namespaces are a MUD concept so prob don't need to include this in the name)